File: Materal.BaseCore/Demo/MBC.Demo.Services/Models/User/QueryUserModel.cs
using Materal.BaseCore.Services;
using Materal.Model;

namespace MBC.Demo.Services.Models.User
{
    /// <summary>
    /// 查询用户模型
    /// </summary>
    public class QueryUserModel : PageRequestModel, IQueryServiceModel
    {
        /// <summary>
        /// 姓名
        /// </summary>
        [Contains]
        public string? Name { get; set; }
        /// <summary>
        /// 账号
        /// </summary>
        [Contains]
        public string? Account { get; set; }
    }
}

File: Level5/RC_OLD/RC.ServerCenter/RC.ServerCenter.Web/src/models/user/QueryUserModel.ts
export default interface QueryUserModel {
    /**
     * 姓名
     */
    Name: string;
    /**
     * 账号
     */
    Account: string;
    /**
     * 页码
     */
    PageIndex: number;
    /**
     * 每页显示数量
     */
    PageSize: number;
}